index.html

```diff
@@ -53,95 +56,108 @@ <h1 class="first-introduction">I'm Raphael.<br /> I am so glad to see you!</h1>
         <!-- Start of Project 1 -->
         <div id="tonic-project">
             <div class="data">
-                <img src="./images/snapshots/Snapshoot Portfolio.png" alt="Tonic project snapshot">
-                <h2 class="project-title">Tonic</h2>
-                <div class="work-nature">
-                    <label class="canopy">CANOPY</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="backend"> Back End Dev</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="year">2015</label>
+                <div class="image-div"><img src="./images/snapshots/Snapshoot Portfolio.png" alt="Tonic project snapshot"></div>
+                <div class="desktop-image-div"><img class="desktop-snapshop" src="./images/desktop/Snapshoot-Desktop1.png" alt="Tonic project snapshot"></div>
+                <div class="project-content-1">
+                    <h2 class="project-title">Tonic</h2>
+                    <div class="work-nature">
+                        <label class="canopy">CANOPY</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="backend"> Back End Dev</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="year">2015</label>
+                    </div>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="tools">
+                        <li><label class="tool-press">HTML</label></li>
+                        <li><label class="tool-press">CSS</label></li>
+                        <li><label class="tool-press">JAVASCRIPT</label></li>
+                    </ul>
+                    <button class="see-project">See Project</button>
                 </div>
-                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="tools">
-                    <li><label class="tool-press">HTML</label></li>
-                    <li><label class="tool-press">CSS</label></li>
-                    <li><label class="tool-press">JAVASCRIPT</label></li>
-                </ul>
-                <button class="see-project">See Project</button>
             </div>
         </div>
```

Comment:
- [ ] As demanded by the project requirements, you must replicate the design as much as possible (e.g., font, colors, images, text, margins), keeping this in mind, kindly consider the following from the image below.
- [ ] Also, kindly add a border around your project cards to match the template.
![image](https://user-images.githubusercontent.com/94551484/229280470-3e4c1a81-4c9e-4e48-bf12-9ac883a247dc.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/229280504-13e6933d-95d2-4d72-831a-28a549a72afc.png)